src/coinbase/coinbase.ts

```diff
@@ -95,6 +98,15 @@ export class Coinbase {
       basePath: basePath,
     });
     const axiosInstance = globalAxios.create();
+    axiosRetry(axiosInstance, {
+      retries: maxNetworkRetries,
+      retryCondition: (error: AxiosError) => {
+        return (
+          error.config?.method?.toUpperCase() === "GET" &&
+          (error.response?.status || 0) in [500, 501, 502, 503, 504]
```

Comment:
we can add more if we need

Reply:
`501` not implemented seems like not worth retrying?

`500` feels borderline, since that would indicate that our service is explicitly throwing that error. However it's likely that there are a number of retryable errors that bubble up to a 500 that we need to squash first.

Reply:
@alex-stone sounds good, I updated.